src/esm.ts

```diff
@@ -56,12 +67,19 @@ export default function hot<P>(
     registration.component.dependencies = Comp.dependencies;
     return false;
   }
-  if (isHot) {
+  if (hot) {
+    const dataPrefix = "solid-refresh-ctx";
```

Comment:
For future references, can we move this up to the top-level? and declared AS_SOME_CONSTANT_VAR?

Reply:
@lxsmnsyc Sure, i'll change that 🙂👍 (probably tomorrow)

Reply:
@lxsmnsyc I changed it :)